docs/node/run-your-node/archive-node.md

```diff
@@ -24,19 +16,17 @@ Archive node requires a pre-existing oasis-node state.
 
 ## Configuration
 
-Archive mode can be enabled by using the `consensus.tendermint.mode` setting.
+Archive mode can be enabled by using the `mode` setting.
 
 :::info
 
-This will configure the given node to act as a consensus archive node.
+This will configure the given node to act as an archive node.
 
 :::
 
 ```yaml
 # other settings omitted ...
-consensus:
-  tendermint:
-    mode: archive
+mode: archive
```

Comment:
TODO: this is actually missing in control status output in 23.x.
Add a note that this is fixed in release xxx.